cfgov/v1/jinja2/v1/includes/molecules/social-media.html

```diff
@@ -70,7 +70,7 @@
 <div class="m-social-media
             m-social-media__{{ 'share' if is_share_view else 'follow' }}">
```

Comment:
```suggestion
            m-social-media--{{ 'share' if is_share_view else 'follow' }}">
```

Reply:
Fixed in https://github.com/cfpb/consumerfinance.gov/pull/8274